### DISCUSSION TITLE
password recovery for Windows

### DISCUSSION THREAD

**Discussion by n3gwg** [antonym/netboot.xyz#799] [General]
netboot.xyz users,

There are a number of password reset ISOs for Windows, are any of them inclusive to the NetBoot environment ISO? I did not see any, but perhaps one of the ISOs there has this functionality?

Stuart

**Comment by chris18890** [comment]
I know Pogostick used to be included, but not sure if it still is, as it was loaded using memdisk & a hosted ISO
